python/aws-lambda/security/tainted-pickle-deserialization.yaml

```diff
@@ -2,11 +2,11 @@ rules:
 - id: tainted-pickle-deserialization
   mode: taint
   pattern-sources:
-  - patterns:
-    - pattern: event
-    - pattern-inside: |
-        def $HANDLER(event, context):
-          ...
+     - patterns:
+         - pattern: |
+             def $HANDLER($EVENT, context):
```

Comment:
Actually, then it would match every random function with 1 or more arguments. I guess this is not yet perfect, but it is the taint source that was used in all other aws-rules, I assumed it has been tested and considered "good enough".